packages/extensions/insights/n8n.manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "@n8n/n8n-extension-insights",
+	"version": "1.0.0",
+	"author": "n8n",
+	"category": "Internal",
+	"minSDKVersion": "0.0.0",
+	"permissions": {
+		"frontend": [],
+		"backend": []
+	},
```

Comment:
It'd be nice if we didn't have to lose type safety, esp. useful for for e.g. `permissions`, `events`, `extends`.

Reply:
I'm working on adding a JSON schema in a separate PR.